include/spl/sys/sysmacros.h

```diff
@@ -115,30 +115,36 @@
 #endif
 
 /* Dtrace probes do not exist in the linux kernel */
+#include <sys/dtrace_probe.h>
 #ifdef DTRACE_PROBE
 #undef  DTRACE_PROBE
 #endif  /* DTRACE_PROBE */
-#define	DTRACE_PROBE(a)					((void)0)
+#define	DTRACE_PROBE(a)					\
+    zfssp_##a()
```

Comment:
Does this stand for "ZFS Static Probe"?  What would you think about `zprobe_` instead, since that highlights the most important part of this (it's a probe point)?

Reply:
Yes, I was thinking ZFS Static Probe but I like zprobe.